src/register.php

```diff
@@ -41,15 +41,13 @@
 			$sth->close();
 
 			// sending email
-			$to = $email;
 			$email_enc = urlencode($email);
 			$subj = "Registration for MARTI dice server";
-			$from = "marti@tripleawarclub.org";
-			$ehead= "From: MARTI<$from>\n";
+			$ehead= "From: MARTI<{${dice::senderEmail}}>\n";
 			$ehead .= "List-Unsubscribe:<$dice->domain/unsubscribe.php?email=$email_enc>\n";
 			$message = "To validate your email click this link: $dice->domain/validate.php?email=$email_enc&val=" . urlencode($validation);
 			$message .= "\n\nTo unsubscribe from this service go to $dice->domain/unsubscribe.php?email=$email_enc";
-			$mailsend= @mail($to, $subj, $message, $ehead, "-f $from -r no-reply@tripleawarclub.org");
+			$mailsend= @mail($email, $subj, $message, $ehead);
```

Comment:
1. I'm assuming that, even with the removal of the explicit `from` parameter, the email is still delivered with a sender because the `mail()` function extracts the sender's address from the `ehead` additional header parameter?
1. I'm assuming it doesn't matter if we set the reply address or not since both `marti@` and `no-reply@` are non-existent mailboxes, and both would result in the same "undeliverable" message if an attempt to reply was made by the recipient?

Reply:
1. I did lookup the documentation of the PHP mail function and the 5th parameter seems to be appended to the underlying sendmail command.
After looking up the sendmail manual, it seems that `-r <email>` is just an obsolete version of `-f <email>` and both set the from mail address, which can also be set in the headers, which is already being done here.

2. Yes

To answer your question about my testing method, I didn't actually spin up a full PHP server to test all of my changes, but I definitely made the changes with a PHP language server, so no syntax error guaranteed and the semantic changes were nothing too crazy. Basically the mail call here is not pretty much the same as the mail call in `dice.class.php`, so that's why I felt no further testing was required.